index.html

```diff
@@ -259,11 +259,12 @@
           </p>
           <ul>
             <li>
-              <code>http://fedora.info/definitions/repository#EmbedResources</code>:
-              Requires a server to include representations of any contained resources in the response.
+              <code>http://www.w3.org/ns/oa#PreferContainedDescriptions</code>:
```

Comment:
The intention of PreferContainedDescriptions is that the representation delivered in response to the request should embed sufficient information about the contained resources to ensure that they are usable by a good number of applications. It doesn't need to be every single triple, but it can't be just the URI (PreferContainedIRIs). That the protocol talks about "annotations" is because of the context, rather than any formal definition -- which is why we went with ContainedDescriptions, not ContainedAnnotations which would have made this use problematic.